lilypond/songs/love-divine-all-loves-excelling/love-divine-all-loves-excelling.ly

```diff
@@ -0,0 +1,55 @@
+\version "2.22.1"
+#(ly:set-option 'relative-includes #t)
+\include "../../lib/hymn-common.ily"
+
+%% TUNE INFO
+\include "../../shared-tunes/hyfrydol-public.ily"
+quarternoteTempo = 120
+
+%% SONG INFO
+title = \titleText "Love Divine, All Loves Excelling"
```

Comment:
My stylistic preference has been to keep capitalization grammatically correct, both in lyrics and the title. 

Regarding capitalize-every-word titles, I think my preference against is mostly arbitrary (what I grew up with), and in the minority. You could probably convince me this is the better way.

Reply:
It's your project, so you should definitely be the one deciding these things, especially for consistency's sake if that's how the other hymns are formatted. I made the change.